### PR TITLE
feat(web/server) Add more options to public shared link

### LINKED ISSUE #1302: [Feature]: ability to share images/albums publicly without sharing GPS exifdata
### Feature detail

I frequently want to share images to people that I don't necessarily want to share my GPS information with. (photos taken at home shared with people on the internet for example). v1.41 added the ability to share albums to people without accounts, which is awesome, but I would like to be able to optionally redact gps metadata from albums that are shared.

### Platform

Web

### PR DESCRIPTION
**Pull request by alextran1502**
This PR adds additional options to the public shared link. Resolves #1302

1. Allow Download
2. Show EXIF

### Default creation form
image

### CODE REVIEW

**Comment by zackpollard** [comment]
Something to consider with this is, if they deny showing exif, we should probably strip the exif from file downloads too, as otherwise that permission is pretty much pointless if they also allow downloading.

**Comment by JVT038** [comment]
Could 'show EXIF' be changed to 'show metadata'?
I don't know if all users will know what "EXIF" means. Changing it to 'metadata' would probably improve clarity, because 'metadata' is more commonly used than 'EXIF'

File: web/src/lib/components/shared-components/gallery-viewer/gallery-viewer.svelte [hunk 1]
```diff
@@ -110,7 +110,8 @@
 {#if isShowAssetViewer}
 	<AssetViewer
 		asset={selectedAsset}
-		publicSharedKey={key}
+		publicSharedKey={sharedLink?.key}
```
Comment (jrasm91) [inline web/src/lib/components/shared-components/gallery-viewer/gallery-viewer.svelte:113]: do we need to pass the whole shared link and the shared link key as two separate props?

**Review by jrasm91 — APPROVED** [review]
Looks good.

**Pull request merged**